_drafts/2020-08-07-g2p.md

```diff
@@ -0,0 +1,625 @@
+---
+layout: post
+title:  "Getting from 'a' to 'b' with g2p"
+author: AidanPine
+categories: []
+tags: [ tutorial, explanation, tech-showcase ]
+image: assets/images/bonjour-g2p.png
+description: "How to get from 'a' to 'b' using the g2p library"
+featured: false
+hidden: false
+---
+
+This blog post describes the background context for a software tool called `g2p`. `g2p` is a tool for systematically converting certain characters[^1] into other ones. Common use cases include converting between a [writing system and the phonetic alphabet](#use-case-1-getting-the-pronunciation-from-a-words-spelling), between two [writing systems for the same language](#use-case-2-a-language-with-multiple-writing-systems), or between a legacy ['font-hacked' writing system and its current Unicode-supported version](#use-case-3-converting-from-legacy-writing-systems).
+
+# What you need to know to understand this post
+
+- To understand the advanced section, you will need to know some [Python](https://www.python.org/)
+
+# Who is involved with this project?
+
+- Maintainer (i.e. the person to bug with questions): [Aidan Pine](https://aidanpine.ca)
+- Lots of [other contributors](https://github.com/roedoejet/g2p/graphs/contributors)
+
+# What are the motivations behind G2P?
+
+There are many reasons why you might want to systematically convert between different characters. Here are a few possible use cases:
+
+### Use Case #1: Getting the pronunciation from a word's spelling
+
+Sometimes you want to convert between a language's writing system (also known as *orthography*) and its pronunciation. "Letters" in a writing system are usually referred to as "graphemes" and their corresponding meaningful sounds are referred to as "phonemes"; hence "g2p" or "grapheme-to-phoneme". 
+
+It gets a little more complicated than that though, because sometimes a grapheme is made of more than one character, as with the *digraph* "th" which can be pronounced *[unvoiced](https://www.thoughtco.com/voiced-and-voiceless-consonants-1212092#:~:text=Voiceless%20consonants%20do%20not%20use,as%20in%20%22thing%22)* as in 'thin' or *[voiced](https://www.thoughtco.com/voiced-and-voiceless-consonants-1212092#:~:text=Voiced%20Consonants,-Your%20vocal%20cords&text=As%20you%20pronounce%20a%20letter,W%2C%20Y%2C%20and%20Z.)* as in 'that'.
+
+The [International Phonetic Alphabet](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) (IPA) is not so ambiguous! In IPA, the 'th' in 'thin' is written as **θ** and the 'th' in 'that' is written as **ð**. Knowing how to convert between the written and spoken form of a language is useful in a variety of computational tasks, but I will describe the usefulness specifically with a project called "ReadAlongs" [below](#readalongs).
```

Comment:
```suggestion
The [International Phonetic Alphabet](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) (IPA) is not so ambiguous! In IPA, the 'th' in 'thin' is written as **θ** and the 'th' in 'that' is written as **ð**. Knowing how to convert between the written and spoken form of a language is useful in a variety of computational tasks, such as aligning written transcriptions with its audio so that it is synced up. I will describe this task further with a project called "ReadAlongs" [in the following post](#readalongs). ** update link, if in following post
```

Reply:
Same thing, I just added this to a commit.